lib/DS18B20/DS18B20.cpp

```diff
@@ -0,0 +1,96 @@
+#include "DS18B20.h"
```

Comment:
Maybe make a new branch that includes logging, *or* add a library for logging & give it compilation parameters for production